cypress-io/cypress
# feat: change test isolation logic to navigate to about:blank at end of tests

### Additional details


With this change, we will alter the test isolation logic to navigate to `about:blank` at the end of a given test rather than at the beginning. In order to facilitate this, we need to change up the logic a little bit to:

* Set duringUserTestExecution to false prior to navigating to about: blank
* Turning off listeners for the window loading (e.g. `window:load`, etc.) so that they will not fire in users tests
* Ensure that in open mode we do not navigate to about:blank on the last test so that the user can interact with the AUT as it was at the end of their test (e.g. for things like studio and selector playground)

### Steps to test


The test coverage for this is fairly solid. You can also test this in open mode to ensure that the about:blank screen does not appear on the last test.

### How has the user experience changed?


n/a

### PR Tasks


- [na] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [na] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
